typescript/src/apis/itask_consumer_api.ts

```diff
@@ -0,0 +1,97 @@
+import {IIdentity} from '@essential-projects/iam_contracts';
+
+import {TaskList} from '../data_models/task/index';
+/**
+ * The ITaskConsumerApi is used to retreive Tasks.
+ */
+export interface ITaskConsumerApi {
+
+  /**
+   * Retrieves a list of all suspended Tasks.
+   *
+   * @async
+   * @param  identity            The requesting users identity.
+   * @returns                    A list of waiting Tasks for the given
+   *                             ProcessModel.
+   *                             Will be empty, if none are available.
+   * @throws {UnauthorizedError} If the given identity does not contain a
+   *                             valid auth token.
+   * @throws {ForbiddenError}    If the user is not allowed to access the
+   *                             ProcessModel.
+   */
+  getAllSuspendedTasks(identity: IIdentity): Promise<TaskList>;
+
+  /**
+   * Retrieves a list of all suspended Tasks belonging to a
+   * specific ProcessModel.
+   *
+   * @async
+   * @param  identity            The requesting users identity.
+   * @param  processModelId      The ID of the ProcessModel for which to
+   *                             retrieve the Tasks.
+   * @returns                    A list of waiting Tasks for the given
+   *                             ProcessModel.
+   *                             Will be empty, if none are available.
+   * @throws {UnauthorizedError} If the given identity does not contain a
+   *                             valid auth token.
+   * @throws {ForbiddenError}    If the user is not allowed to access the
+   *                             ProcessModel.
+   */
+  getTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<TaskList>;
```

Comment:
Hier auch das gleiche wie bei den anderen Contracts.

```suggestion
  getSuspendedTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<TaskList>;
```